admin-dev/themes/new-theme/js/components/form/form-field-disabler.ts

```diff
@@ -45,6 +51,7 @@ export type FormFieldDisablerParams = {
   targetSelector: string | null,
   switchEvent: string | null,
   disableOnMatch: boolean,
+  toggleType: toggleType
```

Comment:
also thinking if its worth renaming whole component? (altough the disabler also doesn't sound that bad, we could even leave toggleType: availability by default)

Reply:
Agreed for the default config, although maybe we could rename it `FormFieldToggler` now? Since it's a next component we still have the occasion to rename it for now